integration/harness_test.go

```diff
@@ -42,8 +42,9 @@ var buildOnce sync.Once
 
 // run an integration test.
 // ftlConfigPath: if FTL_CONFIG should be set for this test, then pass in the relative
-//   path from integration/testdata/go/. e.g. "database/ftl-project.toml"
-func run(t *testing.T, ftlConfigPath string, actions ...action) {
+//
+//	path from integration/testdata/go/. e.g. "database/ftl-project.toml"
```

Comment:
This comment should be changed such that the base directory is relative to where the test is